<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="receiveText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>591</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="interSetting">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>251</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-width: 1;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="interText">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>网络配置</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostIP">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>目前IP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="hostIPText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="hostPost">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>目前Post</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="hostPostText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxConnection">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="clientT">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeConnectButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>490</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>断开所有连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeNowConnectButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>350</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>断开目前连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allSendButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>全体发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
